Sources/Sentry/SentryFramesTracker.m

```diff
@@ -139,15 +144,20 @@ - (void)displayLinkCallback
     CFTimeInterval slowFrameThreshold = 1 / (actualFramesPerSecond - 1);
 
     CFTimeInterval frameDuration = lastFrameTimestamp - self.previousFrameTimestamp;
-    self.previousFrameTimestamp = lastFrameTimestamp;
```

Comment:
This was actually a bug in the existing code: by overwriting `self.previousFrameTimestamp` at this point, that meant that when we call the `recordTimestampStart` method below, the previous timestamp would always be equal to the `lastFrameTimestamp`. Both have the same value.

At the end of this method we're doing `self.previousFrameTimestamp = lastFrameTimestamp;` a second time so this first time is definitely not needed and seems to have been an oversight.

Reply:
Good catch